test/functional/desired.js

```diff
@@ -49,6 +49,7 @@ const GENERIC_CAPS = {
   maxTypingFrequency: 30,
   clearSystemFiles: true,
   showXcodeLog: SHOW_XCODE_LOG,
+  'appium-version': {"appium-url": `bintray:${process.env.APPIUM_SHA}`}
```

Comment:
@dpgraham does Travis know the SHA before running the build? could it be possible to handle this as an environment variable? Just thinking in case we change the capabilities structure, it would give us more flexibility and we wouldnt need to change the code.

Reply:
Should be available as `TRAVIS_COMMIT` (see https://docs.travis-ci.com/user/environment-variables/#convenience-variables)

Reply:
`TRAVIS_COMMIT` would have the sha of `appium-xcuitest-driver`, but it needs the sha for `appium/appium`

Reply:
Good point. Try
```sh
export APPIUM_COMMIT=$(git ls-remote git@github.com:appium/appium.git HEAD | awk '{print $1;}')
```

Reply:
I think that will also have problems because the latest commit might not have a published zip to bintray.